.azure-pipelines/templates/configure_azureml_agent.yml

```diff
@@ -4,6 +4,14 @@ steps:
   inputs:
     versionSpec: '3.9'
 
+- task: Cache@2
```

Comment:
We should add some docs describing how the caching works and the variables used.

Reply:
Added information about this to the root readme, let me know if you think it gives enough context